### PR TITLE
Feature/SearchUser

### PR DESCRIPTION
**Pull request by geoseong**
## Description

- migrate code from `talktalk-rn` project before
- test with sample datas
```
export const fakeUsers: User[] = [
  {
    uid: '1',
    displayName: 'admin',
    thumbURL: 'https://avatars2.githubusercontent.com/u/45788556?s=200&v=4',
    photoURL: 'https://avatars2.githubusercontent.com/u/45788556?s=200&v=4',
    statusMsg: 'online',
    online: true,
    // created: new Date(),
    // updated: new Date(),
  },
  ...
]
```
- Make the Image Component in 'ProfileModal' styled same as 'UserListItem'
- make eslint in vscode work with .eslintrc

## Related Issues
#13 

## Tests

- when friend name typed in TextInput: (onTxtChanged -> onSearch) and (renderItem) and (keyExtractor)
- see Animation is working well

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/hackatalk-mobile/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 module.exports = {
   root: true,
   extends: '@dooboo/eslint-config',
+  rules: {
+    'jsx-quotes': ['error', 'prefer-double'],
```
Comment (hyochan) [inline .eslintrc.js]: Are you sure we need this? This is already defined in `@dooboo/eslint-config`.

File: src/components/screen/SearchUser.tsx [hunk 2]
```diff
@@ -1,30 +1,244 @@
...
+
+import EmptyListItem from '../shared/EmptyListItem';
+import { User } from '../../types';
+import UserListItem from '../shared/UserListItem';
+import { getString } from '../../../STRINGS';
 import styled from 'styled-components/native';
+import { useStateValue } from '../../contexts';
 
-const Container = styled.View`
+export const userSampleData: User[] = [
```
Comment (hyochan) [inline src/components/screen/SearchUser.tsx]: The naming would be better if you use `fakeUsers`. Just to confirm this data is temporary.

File: .eslintrc.js (hunk 1 above)
Comment (geoseong) [reply .eslintrc.js]: really? `jsx-quotes` errors happened in circleci test. so after I added this rules, errors disappeared.

**Comment by geoseong** [comment]
> image
> 
> I still see the error in ci.

it is hard to check clear in local. running `yarn test` doesn't return any errors...

**Comment by hyochan** [comment]
@geoseong Nono that isn't `yarn test` error. That is `tsc`. You should run it.

File: .eslintrc.js (hunk 1 above)
Comment (hyochan) [reply .eslintrc.js]: This is strange.. It is already defined in our rule [here](https://github.com/dooboolab/eslint-config/blob/8b62bb9ec7dcb83a1f2c5310aa41d600f42ff528/index.js#L35).

Comment (geoseong) [reply .eslintrc.js]: I removed rules again in the commit of [2dca9bf](https://github.com/dooboolab/hackatalk-mobile/pull/23/commits/2dca9bff486577b172d48ad43c23072ce051c59c). and it passed... it's strange..

**Comment by geoseong** [comment]
@hyochan I've done what you've pointed out and complemented about settings

**Comment by hyochan** [comment]
@all-contributors please add @geoseong for code, test.

**Pull request merged**